src/App.tsx

```diff
@@ -1,21 +1,71 @@
-import './App.css'
 import CarForm from './components/carForm'
 import { useState } from 'react'
 import { Car } from './types'
+import VCV from '../lib/vcv'
 import CarList from './components/carList'
+import CarEditDialog from './components/carEditDialog'
+import CarHistory from './components/carsHistory'
+import { Box, Button, Container, Divider, ThemeProvider} from '@mui/material'
+
+import themes from './themes'
 
 function App() {
-  const [cars, setCars] = useState<Car[]>([]) // Car array state
+	const { state : cars, 
+					setValue : setCars, 
+					revertTo : revertCommit, 
+					history : commitHistory,
+					index: commitIndex } = VCV<Car[]>([]) // Car array state
+	const [themeIndex, setThemeIndex] = useState(0)
```

Comment:
Please, fix spacings here and in all other places. Are you using `prettier-formatter` to format code on save?